packages/io/src/nervos.chain-source.ts

```diff
@@ -1,11 +1,4 @@
-/**
- * @module src/chain-source
- * @description
- * This module defines the chain source for the application.
- * A chain source could be a CKB indexer.
- */
-
-import type { ChainSource } from '@ckb-js/kuai-io/lib/types'
+import type { ChainSource } from './types'
```

Comment:
Why should we remove the comments?

Reply:
> Why should we remove the comments?

I saw that there is no other file in this package has this comment, so I remove it. Should I add it back?

Reply:
Of cause, it is used to build docs with [typedoc](https://typedoc.org/)

Reply:
> Of cause, it is used to build docs with [typedoc](https://typedoc.org/)

OK, but I have changed its directory, from mvp to io package, isn't it ok?

Reply:
Oh, if so, we can remove the comments, `typedoc` will not build docs for `kuai` library.